Birthday-2024-Project/Scripts/GridLogic.gd

```diff
@@ -4,7 +4,7 @@ class_name GridLogic
 const MAX_HEIGHT : int = 4 #doubled for negative
 const MAX_WIDTH : int = 5 #doubled for negative
 const TILE_ATLAS_OPEN = Vector2i(1, 0)
-const TILE_ATLAS_OCCUPIED = Vector2i(1, 0)
+const TILE_ATLAS_OCCUPIED = Vector2i(2, 0)
```

Comment:
what is the purpose of this change?

Reply:
It changes which tile is used in the tileset. This changes the "occupied" tiles back to using the 3rd tile